.changeset/sweet-garlics-eat.md

```diff
@@ -0,0 +1,7 @@
+---
+"@justeattakeaway/pie-tag": minor
+"pie-storybook": patch
+"pie-docs": patch
+---
+
+[Added] - new variants `brand-03`, `brand-04` and `brand-06` to the tag component
```

Comment:
Sorry to bring up naming again 🙈 I personally think these should be called something like `brand-alt-1`, `brand-alt-2` or something like that. Otherwise, the names are still pretty tightly tied to the token names, rather than being abstracted slightly.

Please don't hate me for bringing this up though 😆

Reply:
let's take this offline and will update the thread with the outcome 🤝

Reply:
Resolved offline – sticking with what it is currently